src/components/__tests__/home.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Home from "../home";
+​
+test("renders home page content", () => {
+  render(<Home />);
+​
```

Comment:
- [ ] This test is failing. kindly consider adding a `describe` block here. that could be the reason for the failed test.